Improve startup action warning in Restore page (#180)
hazelcast/hazelcast-platform-operator-docs: docs/modules/ROOT/pages/backup-restore.adoc

To recover a cluster that has Persistence enabled after a cluster-wide restart, you can force a cluster to delete their persistence stores when one or more members fail to restart by triggering Force Start. Or you can force a cluster to start without some members by triggering Partial Start

> WARNING: Performing a force-start may result in the loss of all persistent data in the cluster. Additionally, executing a partial-start may lead to the loss of some persistent data.

Reviewer: ```suggestion
[WARNING]
====
* The cluster loses all persisted data after a force-start.
* The cluster loses some persisted data after a partial-start.
====
```

Author: `force-start`: In the event that any member fails to start properly, this action will result in the deletion of all persisted data. However, if all members successfully start, all persisted data will remain intact.

`partial-start`: If any member fail to start properly, only the persisted data associated with that specific member will be deleted, while data from other members will remain unaffected.

Reviewer: Thanks. I'd like to see that detail, it is clearer and more precise than simply stating the possibility of persisted data loss.
